index.html

```diff
@@ -367,18 +347,22 @@ <h2 class="about-heading">
             <li>
               <div class="lang-correction">
                 <p class="lang-text">Frameworks</p>
-                <button type="button" class="dwn-btn"><i class="fa fa-angle-right" aria-hidden="true"></i></button>
+                <button type="button" class="dwn-btn">
+                  <i class="fa fa-angle-right" aria-hidden="true"></i>
+                </button>
               </div>
-              <hr class="line"/>
+              <hr class="line" />
               <!-- <div><img src="/images/SeparatorBottom.svg" class="seperator"></div> -->
             </li>
 
-             <li>
+            <li>
               <div class="lang-correction">
                 <p class="lang-text">Skills</p>
```

Comment:
- Optional
Since this is not in today's project requirements, however, I strongly recommend you fix it.
This one element is breaking the layout of your application in mobile view since it is scaling horizontally more than others. Consider fixing it.
![image](https://user-images.githubusercontent.com/33268927/181474411-41ccf4b3-3545-447b-8bc3-3d9daa052857.png)